pool/pool.go

```diff
@@ -1371,7 +1398,12 @@ func (p *Pool) PutContainer(ctx context.Context, prm PrmContainerPut) (*cid.ID,
 		return nil, err
 	}
 
-	return res.ID(), nil
+	waitParams := DefaultWaitParams()
+	if prm.waitParamsSet {
+		waitParams = &prm.waitParams
```

Comment:
(same for `DeleteContainer` and `SetEACL`)

If `waitParamsSet` is true we do redundant `DefaultWaitParams`. I suggest to:

- make default parameters unexported
- change condition with smth like that:
```
if !prm.waitParamsSet {
  prm.waitParams.setDefaults() // unexported method
}
```

Docs suggestion:
```
// PutContainer sends request to save container in NeoFS and and waits for the operation to complete.
//
// Waiting parameters can be specified using SetWaitParams. If not called, defaults are used:
//   polling interval: 5s
//   waiting timeout: 120s
```